Draft/draft-ietf-rtcweb-stun-consent-freshness.xml

```diff
@@ -378,16 +391,25 @@
       an API hook that generates an event when consent has expired for a given
       5-tuple, meaning that transmission of data has ceased. This could
       indicate what application data is affected, such as media or data
-      channels.</t>
+      channels. The circuit breaker algorithm discussed in section 7.1 of
+      <xref target="I-D.ietf-rtcweb-rtp-usage"></xref> could also be one of
+      the reasons for ceasing transmission of media and to notify the
+      application about the loss of ability to transmit data.</t>
```

Comment:
Why did you agree to including mention of circuit breakers?  They are irrelevant here.  And potentially confusing.